schemas/ephys/Ephys Rig Schema.json

```diff
@@ -0,0 +1,310 @@
+{
+	"$schema": "https://json-schema.org/draft/2020-12/schema",
+	"type": "object",
+	"properties": {
+		"version": {
+			"type": "string",
+			"const": "0.1.0",
+			"description": "schema version",
+			"title": "Version"
+		},
+		"rig_id": {
+			"type": "string",
+			"const": "0_0_0",
+			"description": "room_stim apparatus_version",
+			"title": "Rig ID"
+		},
+		"devices": {
+			"type": "object",
+			"properties": {
+				"probes": {
+					"type": "array",
+					"items": { "$ref": "#/$defs/probe" },
+					"uniqueItems": true,
+					"title": "Probes"
+				},
+				"cameras": {
+					"type": "array",
+					"items": {"$ref": "#/$defs/camera" },
+					"uniqueItems": true,
+					"title": "Cameras"
+				},
+				"lasers": {
+					"type": "array",
+					"items": {"$ref": "#/$defs/laser" },
+					"uniqueItems": true,
+					"title": "Lasers"
+				},
```

Comment:
should be plural "monitors"?

Reply:
yeah, I'll do it. concerns: a) never have more than one but that's fine. b) I put this in as an example for the kind of device information we need. If we're really using visual stimuli, we need to really evaluate if this has everything it needs.
I wonder if we should take it out for now and figure out what are the devices that we're using.

Reply:
Got it. I think it's fine to leave it there but have it underspecified. e.g. just a protocol and a notes section or whatever.